cmd/global-heal.go

```diff
@@ -342,11 +346,7 @@ func (er *erasureObjects) healErasureSet(ctx context.Context, buckets []string,
 				}
 				bgSeq.logHeal(madmin.HealItemObject)
 
-				select {
-				case <-ctx.Done():
-					return
-				case results <- result:
-				}
+				send(result)
```

Comment:
This no longer returns when ctx.Done. Is that caught elsewhere?

Maybe return a bool from `send`.

Reply:
oh yes :-)